project_task_materials/models/project.py

```diff
@@ -1,40 +1,31 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2012 - 2013 Daniel Reis
-#    Copyright (C) 2015 - Antiun Ingeniería S.L. - Sergio Teruel
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
-from openerp import models, fields
+# © 2012 - 2013 Daniel Reis
+# © 2015 - Antiun Ingeniería S.L. - Sergio Teruel
+# © 2017 - Rigoberto Martínez <rigo1985@gmail.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
 
 
 class Task(models.Model):
     _inherit = "project.task"
-    material_ids = fields.One2many(
-        comodel_name='project.task.materials', inverse_name='task_id',
-        string='Materials used')
+
+    material_ids = fields.One2many(comodel_name='project.task.materials',
+                                   inverse_name='task_id',
+                                   string='Materials used')
```

Comment:
Please avoid code style only changes.
Both ways are PEP8 compliant, but when so IMO the original author's preference should be respected.